lib/plugins/inspec-streaming-reporter-progress-bar/inspec-streaming-reporter-progress-bar.gemspec

```diff
@@ -0,0 +1,9 @@
+# .gemspec file is added to add plugin details
+# These specs are used in plugin list and search command
+
+Gem::Specification.new do |spec|
+  spec.name          = "inspec-streaming-reporter-progress-bar"
+  spec.summary       = "Progress bar streaming reporter plugin"
```

Comment:
This doesn't add much that isn't already known by the name. Try something like "Displays a real-time progress bar and control title as output."